README.md

```diff
@@ -10,6 +10,8 @@ Build/Runs your crystal application, watches files, and rebuilds/reruns app on f
 
 ## Installation
 
+### Installation using ruby script
```

Comment:
I'd opt to rename this from **Installation using ruby script** to **Quick Installation**.